package.json

```diff
@@ -26,21 +26,23 @@
     "@types/yargs": "^15.0.10",
     "aws-sdk": "^2.923.0",
     "dotenv": "^8.2.0",
-    "serverless": "^2.72.3",
-    "serverless-bundle": "^4.3.1",
-    "serverless-dotenv-plugin": "^3.0.0",
+    "serverless": "^3.12.0",
+    "serverless-bundle": "^5.3.0",
+    "serverless-dotenv-plugin": "^3.12.2",
     "serverless-iam-helper": "CMSgov/serverless-iam-helper",
     "serverless-idempotency-helper": "CMSgov/serverless-idempotency-helper",
-    "serverless-offline": "^6.8.0",
+    "serverless-offline": "^8.5.0",
     "serverless-online": "CMSgov/serverless-online",
     "serverless-plugin-scripts": "^1.0.2",
-    "serverless-plugin-warmup": "^5.3.1",
+    "serverless-plugin-warmup": "^7.1.0",
     "serverless-s3-bucket-helper": "CMSgov/serverless-s3-bucket-helper#0.1.1",
-    "serverless-stack-termination-protection": "^1.0.4",
+    "serverless-stack-termination-protection": "^2.0.0",
     "typescript": "^4.0.5",
     "yargs": "^16.1.1"
   },
-  "dependencies": {},
+  "dependencies": {
+    "global": "^4.4.0"
```

Comment:
Was this actually needed?  I'm surprised to see it added

Reply:
I'm not sure. I can take it out and see if anything crashes